aait/backend/group-2A/Application/Features/Comment/Handlers/Commands/CreateCommentCommandHandler.cs

```diff
@@ -17,19 +19,27 @@ public CreateCommentCommandHandler(IUnitOfWork unitOfWork, IMapper mapper)
         _unitOfWork = unitOfWork;
     }
 
-    public async Task<int> Handle(CreateCommentCommand request, CancellationToken cancellationToken)
+    public async Task<BaseCommandResponse> Handle(CreateCommentCommand request, CancellationToken cancellationToken)
     {
+        var response = new BaseCommandResponse();
         var validator = new CreateCommentDtoValidator(_unitOfWork.postRepository, _unitOfWork.userRepository);
         var validationResult = await validator.ValidateAsync(request.CommentDto);
 
         if (!validationResult.IsValid)
         {
-            throw new ValidationException(validationResult.Errors);
```

Comment:
if there is validation error don't throw validation exception : remove the following line . because you don't need it
throw new ValidationException(validationResult.Errors);